aya/src/programs/cgroup_skb.rs

```diff
@@ -56,6 +58,7 @@ use crate::{
 pub struct CgroupSkb {
     pub(crate) data: ProgramData<CgroupSkbLink>,
     pub(crate) expected_attach_type: Option<CgroupSkbAttachType>,
+    pub(crate) path: Option<PathBuf>,
```

Comment:
It seems you've added this to all the program structs - doesn't it belong to
ProgramData then? (ProgramData is used for fields common to all structs)

Reply:
Done.